documentation/general/deployment.md

```diff
@@ -148,13 +148,13 @@ release for an application. Publishing a release automatically tags the latest
 docker image for the application and opens a PR to add the changelog to the
 [documentation site's changelog directory](https://docs.openverse.org/changelogs/index.html).
 This needs to be manually approved and merged by maintainers. The person who
-triggers the release app workflow is pinged in the PR description to help with
-the visibility of the PR.
+publishes the release is pinged in the PR description to help with the
+visibility of the PR.
 
 For the API and frontend, publishing the release also triggers an automated
-production deployment. For the catalog and ingestion server, however,
-maintainers must manually deploy the changes to production using the Terraform
-deployment process.
+production deployment using the release app workflow. For the catalog and
```

Comment:
```suggestion
production deployment using the deploy app workflow. For the catalog and
```

This might provide even more clarity, and also better matches the `deploy-{environment}-{service}.yml` naming of the workflows. 

We could consider linking to those files: https://github.com/WordPress/openverse-infrastructure/tree/main/.github/workflows for additional clarity, but not necessary.